src/hash/clu_alg_hash.c

```diff
@@ -48,10 +67,31 @@ int wolfCLU_md5Setup(int argc, char** argv)
         }
     }
 
+    /* sets default size of algorithm */
+#ifndef NO_MD5
+    if (XSTRNCMP(alg, "md5", 3) == 0)
```

Comment:
Lets avoid the extra string compares and use something like WOLFCLU_CERT_SHA256 from the main function when it first parses the arguments. That then gets passed into this hashing function. Then a switch statement on that input for the hash size/type.

Reply:
Done.